### PR TITLE
Remove unnecessary comments from markdown files

### PR DESCRIPTION
**Pull request by alex-page**
- [x] Replaced `<!-- usagelist` and `<!-- usageblock` with `<!-- dodont`
- [x] Simplified CSS and logic for `<!-- dodont`
- [x] Removed `<!-- keywords: `
- [x] Removed `<!-- showcasecontent`
- [x] Removed `<!-- centeredcontent`
- [x] Removed `<!-- hint`
- [x] Removed 80 character line break from markdown files

### CODE REVIEW

**Comment by martenbjork** [comment]
>  Replaced <!-- usagelist and <!-- usageblock with <!-- dodont

Love it.

**Comment by alex-page** [comment]
@martenbjork if you had any feedback i'll do another PR. I needed to ship this to run automations for old -> new.

File: polaris.shopify.com/content/foundations/content/product-content.md [hunk 1]
```diff
@@ -29,98 +29,42 @@ keywords:
...
 
-Our content standards will help you understand how to think strategically about
-the language in your products and apps. They’ll also give you clear, tactical
-suggestions designed to help you use language to craft better experiences.
-
-[Each component](/components/get-started) also includes content guidelines about
-how to write for specific interface elements.
-
----
+Our content standards will help you understand how to think strategically about the language in your products and apps. They’ll also give you clear, tactical thegestions designed to help you use language to craft better experiences. suggestionsnent](/components/get-started) also includes content guidelines about suggestionsnentr specific interface elements. how how
```
Comment (martenbjork) [inline polaris.shopify.com/content/foundations/content/product-content.md:34]: Broken link

File: polaris.shopify.com/content/foundations/content/product-content.md [hunk 2]
```diff
@@ -29,98 +29,42 @@ keywords:
...
-to learn about who they are, what they need, and the language they use.
-
----
-
-## Use plain language
-
-<!-- keywords: literacy, english as a first language, english as a second language, esl, short sentences, using headings, using bullets, scannable, jargon, idioms, terminology, industry standard, repeated words, small screens -->
+Not everyone is a confident writer, but everyone can improve their content by making sure it responds to the needs of their audience. Keep in mind that rea makingn Shopify every day to run their businesses. The product, feature, or app you’re building can make a big difference to the people using it. Take some t making relyse plain language youpify merchants are busy people who may be running their online business in youition to having a full-time job, managing their family life, and doing a million other things. They’re also located all over the world, have varying youels of literacy, and some m toopify using plain language doesn’t mean dumbing content down. It’s millionaking sure language is straightforward and communicates concepts as millionntly as possible. As a benchmark, we consider plain language to be a U addition
```
Comment (martenbjork) [inline polaris.shopify.com/content/foundations/content/product-content.md:38]: Corrupt sentence

File: polaris.shopify.com/content/foundations/content/alternative-text.md [hunk 3]
```diff
@@ -57,81 +52,64 @@ Alt text should:
...
+If space constraints require you to write calls to action without nouns, like “learn more” and “apply now”, give further indication of where merchants will be sent after they select.
 
-<!-- prettier-ignore -->
 ```html
-<a href="{cta-url}" aria-label="Learn more about opening an online store with Shopify">Learn more</a>
+<a
+  href="{cta-url}"
+  aria-label="Learn more about opening an online store with Shopify"
+  >Learn more</a
+>
```
Comment (martenbjork) [inline polaris.shopify.com/content/foundations/content/alternative-text.md:106]: Weird formatting

**Review by martenbjork — COMMENTED** [review]
@alex-page This PR broke the content on a few pages. I think we should go through it line by line and make sure everything looks good. Some things I spotted when I did a quick scan:

File: polaris.shopify.com/content/foundations/content/product-content.md (hunk 1 above)
Comment (martenbjork) [reply polaris.shopify.com/content/foundations/content/product-content.md:34]: > suggestionsnent

**Pull request merged**